civis/io/_tables.py

```diff
@@ -973,3 +973,12 @@ def callback(future):
             return _download_file(url, filename, headers, compression)
 
     return callback
+
+
+def _robust_schema_table_split(table):
+    schema_name_tup = next(csv.reader(StringIO(table), delimiter="."))
```

Comment:
Wow, this is a great find! It might be worth explicitly setting some other parameters instead of leaving them to the defaults. It looks like `doublequote` and `quotechar` are the two most important.